docs/Customer Secrets EA.md

```diff
@@ -0,0 +1,76 @@
+# Customer Secrets EA
+
+### Prerequisites
+
+***Terminus***
+
+Have terminus [installed](https://pantheon.io/docs/terminus/install) and working
+
+***Terminus Plugin (all platforms)***
+
+1. `terminus plugin:search secret` will yield more than one secret manager. You want the one with the word MANAGER in the title. 
+
+1. `terminus plugin:install pantheon-systems/terminus-secrets-manager-plugin` The other secret “manager” is legacy code and unsupported. [Jira ticket to update that listing with “deprecated”](https://getpantheon.atlassian.net/browse/CMS-962).
+
+1. `terminus` without any arguments will list all the commands. You should now see “secret” commands in that list.
+
+### Steps
+
+***Before***
+
+`terminus auth:login`
+
+***To create a fresh site for testing***
+
+1. Set the upstream to the ID of our Drupal 9 composer-based upstream: 
+
+	 `export SITE_UPSTREAM_ID=bde48795-b16d-443f-af01-8b1790caa1af`
+
+1. `export SITE_NAME=$USER-secrets-testing`
+
+1. `terminus site:create ${SITE_NAME} “${USER} Secrets Testing”  ${SITE_UPSTREAM_ID}`
+
+1.  `terminus local:clone ${SITE_NAME}`
+
+1. Set secrets using one of the methods shown below. Changing a file and pushing the code to development will trigger a composer run.
+
+***Github token authentication***
+
+`terminus secret:set ${SITE_NAME} \
+   github-oauth.github.com ${GITHUB_TOKEN} \
+   --type=composer --scope user --scope ic`
+
+The Github token needs all of the "repo" permissions:![image](https://user-images.githubusercontent.com/87093053/191616923-67732035-08aa-41c3-9a69-4d954ca02560.png)
+
+
+***HTTP basic authentication***
+
+For multiple private repositories, you will need to create a COMPOSER_AUTH json and make it available via the COMPOSER_AUTH environment variable.
+
+Composer has the ability to read private repository access information from the environment variable: COMPOSER_AUTH. The COMPOSER_AUTH variables has to be in a [specific JSON format](https://doc.codingdict.com/composer/doc/articles/http-basic-authentication.html). 
+
+That format example is here:
+
+```bash
+read -e COMPOSER_AUTH_JSON <<< {
+    "http-basic": {
+        "github-oauth.github.com": {
```

Comment:
I think that `github-oauth.github.com` is out of place here.

Reply:
you think it should be GitHub.com ?